dev/tasks/conda-recipes/arrow-cpp/meta.yaml

```diff
@@ -383,33 +377,31 @@ outputs:
         # skip tests that raise SIGINT and crash the test suite
         {% set tests_to_skip = tests_to_skip + " or (test_csv and test_cancellation)" %}  # [linux]
         {% set tests_to_skip = tests_to_skip + " or (test_flight and test_interrupt)" %}  # [linux]
-        # tests that may crash the agent due to out-of-bound memory writes or other risky stuff
-        {% set tests_to_skip = tests_to_skip + " or test_debug_memory_pool" %}            # [aarch64 or ppc64le]
-        # cannot pass -D_LIBCPP_DISABLE_AVAILABILITY to test suite for our older macos sdk
-        {% set tests_to_skip = tests_to_skip + " or test_cpp_extension_in_python" %}      # [osx]
         # skip tests that make invalid(-for-conda) assumptions about the compilers setup
         {% set tests_to_skip = tests_to_skip + " or test_cython_api" %}                   # [unix]
         {% set tests_to_skip = tests_to_skip + " or test_visit_strings" %}                # [unix]
         # skip tests that cannot succeed in emulation
         {% set tests_to_skip = tests_to_skip + " or test_debug_memory_pool_disabled" %}   # [aarch64 or ppc64le]
         {% set tests_to_skip = tests_to_skip + " or test_env_var_io_thread_count" %}      # [aarch64 or ppc64le]
+        # XMinioInvalidObjectName on osx/win: "Object name contains unsupported characters"
+        {% set tests_to_skip = tests_to_skip + " or test_write_to_dataset_with_partitions_s3fs" %}  # [osx or win]
         # vvvvvvv TESTS THAT SHOULDN'T HAVE TO BE SKIPPED vvvvvvv
-        {% set tests_to_skip = tests_to_skip + " or test_extension_to_pandas_storage_type" %}
-        # segfaults on OSX: to investigate ASAP
-        {% set tests_to_skip = tests_to_skip + " or test_flight" %}                       # [osx]
+        # currently broken
+        {% set tests_to_skip = tests_to_skip + " or test_fastparquet_cross_compatibility" %}
+        # new fsspec changed behaviour, see https://github.com/apache/arrow/issues/37555
+        {% set tests_to_skip = tests_to_skip + " or test_get_file_info_with_selector" %}
+        # problems with minio
+        {% set tests_to_skip = tests_to_skip + " or (test_delete_dir and S3FileSystem)" %}
+        {% set tests_to_skip = tests_to_skip + " or (test_delete_dir_contents and S3FileSystem)" %}
+        {% set tests_to_skip = tests_to_skip + " or (test_get_file_info and S3FileSystem)" %}
+        {% set tests_to_skip = tests_to_skip + " or (test_move_directory and S3FileSystem)" %}
```

Comment:
In the broadest sense, this also falls under #37692, however, since those tests were never run in the conda recipes so far (because `minio` hadn't been available until now), it might also be that it's just stumbling over some edge cases of  the manual `minio` setup here.